package.json

```diff
@@ -57,7 +57,7 @@
     ]
   },
   "dependencies": {
-    "3d-view": "^2.0.0",
+    "3d-view-controls": "git://github.com/archmoj/3d-view-controls.git#fe3c77b807618c0d51f4c50a691e2af0486bfda9",
```

Comment:
https://github.com/mikolalysenko/3d-view-controls/compare/master...archmoj:ortho-view5

Reply:
Do we have write and publish access to that module?

Reply:
How hard (or inconvenient) would it be to remove our dependency on `3d-view-controls` and bring all the camera logic within plotly.js?

Reply:
Unfortunately no!

Reply:
> How hard (or inconvenient) would it be to remove our dependency on `3d-view-controls` and bring all the camera logic within plotly.js?

The only issue is that `gl-plot3d` needs to access `3d-view-controls` too.

Reply:
Thanks for the info!

Could you then move the camera code to `gl-plot3d` instead?

Reply:
Good idea.

Reply:
Done in https://github.com/plotly/plotly.js/pull/3550/commits/e1dff30921721df1ca37140179ea276602cf0b34.